File: WebAPI2/StoryBoard/Dto/StoryDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Dto;

public class StoryDto
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public DateTime CreationDate { get; set; }
    public int IdSprint { get; set; }
}

File: UI/StoryBoard/src/app/dtos/story-dto/story-dto.ts
export class StoryDto {
    id : number = 0;
    name : string = "";
    description : string = "";
    creationDate : Date = new Date;
    idSprint: number = 0;

    static createFromFormValues(formValues: any): StoryDto {
        const story = new StoryDto();
        story.name = formValues.name;
        story.description = formValues.description;
        story.creationDate = formValues.creationDate;
        story.idSprint = formValues.idSprint;
        return story;
    }
}